package.json

```diff
@@ -36,8 +36,8 @@
     "eslint-plugin-import": "^1.15.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.2.1",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "file-loader": "^0.9.0",
+    "extract-text-webpack-plugin": "grrowl/extract-text-webpack-plugin#order-undefined-error",
```

Comment:
This fork allows ignoring the undefined order of our CSS modules. We should fix this at its root cause but it's not entirely clear how to do that at the moment: https://github.com/webpack-contrib/extract-text-webpack-plugin/pull/166